docker/fullnode/README.md

```diff
@@ -30,6 +29,29 @@ Get the latest version of the Sui DevNet genesis [genesis.blob](https://github.c
 ```wget https://github.com/MystenLabs/sui-genesis/raw/main/devnet/genesis.blob```
 
 
+### docker compose 
```

Comment:
They use Docker Compose on their site, should match casing.

Reply:
ok